### DISCUSSION TITLE
How do I contribute to this repo?

### DISCUSSION THREAD

**Discussion by chiragxarora** [Vanshika2063/Leetcode-Questions#147] [Q&A] [answered]
Hi @Vanshika2063 
How does this repo work? Can I add questions on my own or you have your questions listed and contributors need to do only those?

**Comment by Vanshika2063** [comment, marked as answer]
look [CONTRIBUTING.md](https://github.com/Vanshika2063/Leetcode-Questions/blob/main/CONTRIBUTING.md) file
